src/style.css

```diff
@@ -2,0 +1,114 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+:root {
+  --main-color: #3d4e61;
+}
+
+html {
+  font-family: Avenir, Arial, sans-serif;
+  color: var(--main-color);
+  font-weight: 300;
+}
+
+.search {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search--big {
+  height: 70px;
+  width: 100%;
+}
+
+.search--small {
+  height: 42px;
+  width: 100%;
+}
+
+.search__label {
+  position: relative;
+  width: 100%;
+  display: block;
+}
+
+.search__label::after {
+  content: '';
+  position: absolute;
+  display: block;
+
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.search__label--big::after {
+  top: 25px;
+  left: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.search__label--small::after {
+  top: 15px;
+  left: 13px;
+  width: 11px;
+  height: 11px;
+}
+
+.search__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
```

Comment:
This is a code quality issue. You've defined the border style for `.search__input` here and then override it in the `.search__input:focus` selector. It's better to define the default and focus styles in the same place to make the code easier to read and maintain.